### PR TITLE
Entity print enabled but hidden by default

### LINKED ISSUE #3204: Consider adding entity print stuff to default config (but off)
## Issue

Anytime default entity view config is changed and Admissions hasn't been updated, configuration imports fail.

https://github.com/uiowa/uiowa/pull/3048#issuecomment-779490809
https://github.com/uiowa/uiowa/pull/3154
https://github.com/uiowa/uiowa/pull/3331#issuecomment-814432897
https://github.com/uiowa/uiowa/pull/3928#issuecomment-896994724
https://github.com/uiowa/uiowa/pull/4111

## Proposed Solution

- Update the affected entity view configuration to include but hide the entity print stuff similar to how we handled the search api thing. https://github.com/uiowa/uiowa/pull/2092
- Check permissions on routes

### PR DESCRIPTION
**Pull request by e-marie-w**
Fixes #3204 

# How to test
- Sync a site, ensure that there are no issues, check basic site functionality.
  - Test that routes like `/admin/config/content/entityprint` are only available to admins. 
- Sync admissions and test as webmaster:
  - https://admissions.uiowa.ddev.site/kiosk
  - https://admissions.uiowa.ddev.site/areas-of-study/print-links
- Sync a v2 site and check that entity print stuff is hidden on `/admin/structure/paragraphs_type/section/display`.

### CODE REVIEW

**Comment by makurj** [comment]
Follow the steps here https://github.com/uiowa/uiowa/pull/5611 for further work, testing with main being merged in.

**Comment by bspeare** [comment]
I think this is ready for review.   Let me know if there are more steps to add to the testing instructions.

**Comment by pyrello** [comment]
I'm going to test this.

**Comment by pyrello** [comment]
I don't think this is ready for review yet. Since we are enabling `entity_print` as a baseline dependency, then all feature and site splits must be updated to reflect this change. I'm only seeing changes to Admission's site split.

**Comment by bspeare** [comment]
@pyrello can you elaborate a little on what we need to do there with the splits and features?  How does that differ with what @joewhitsitt was saying here?  https://github.com/uiowa/uiowa/issues/3204#issuecomment-1265663502

**Comment by joewhitsitt** [comment]
i didn't think beyond the default content types. for example the event content type is a complete split item and will need entity print added to it

**Comment by bspeare** [comment]
@joewhitsitt thanks! That clears it up.

File: config/default/core.extension.yml [hunk 1]
```diff
@@ -38,6 +38,8 @@ module:
   entity_browser: 0
   entity_browser_entity_form: 0
   entity_embed: 0
+  entity_print: 0
+  entity_print_views: 0
```
Comment (pyrello) [inline config/default/core.extension.yml]: Do we want `entity_print_views` enabled for default too? I'm guessing that this isn't necessary for what we are trying to accomplish with this PR.

**Comment by pyrello** [comment]
I have an open question above, but otherwise I think this is ready for review again.

File: config/default/core.extension.yml (hunk 1 above)
Comment (joewhitsitt) [reply config/default/core.extension.yml]: if we can avoid turning on the module to rid us of the admissions issue, i'd vote for that

**Review by pyrello — APPROVED** [review]
I reviewed all changed files. Synced default, admissions, and HMB to check everything. I think this is good.

**Pull request merged**